templates/navigation/breadcrumb.html.twig

```diff
@@ -10,8 +10,21 @@
  * @see ./core/themes/stable/templates/navigation/breadcrumb.html.twig
  */
 #}
+
 {% if segments %}
```

Comment:
Because you got rid of the segments variable in the preprocess function, this code is never called. You also already format the links in the preprocess so there is no need for you to re-do it in the template.

Reply:
After investigation, we could remove this template.